pom.xml

```diff
@@ -9,20 +9,18 @@
   <version>1.0-SNAPSHOT</version>
 
   <name>Java23</name>
-  <!-- FIXME change it to the project's website -->
-  <url>http://www.example.com</url>
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
```

Comment:
Generalnie tych plików tutaj nie powinno być. W zakładce:

![image](https://user-images.githubusercontent.com/17989854/130434658-31f81e15-7505-40fe-be1d-b11332d2b1a2.png)

File changed widzisz jakie zmiany są wysyłane w tym pull requeście. Tutaj tak na prawdę powinno być tylko README.md i nic więcej.

Z jakiegoś powodu masz tutaj więcej zedytowanych plików. Widzę, że na masterze (branch master) masz już jakieś plki. Usuń je proszę całkowicie. Nie powinny tam się znajdować. Możesz to zrobić na tym pull requeście, aby nie robić zamieszania czyli w skrócie 

https://github.com/Sticky5/Java23

wszystkie te pliki do usunięcia, a dochodzi tylko README.md